baseapp/msg_service_router.go

```diff
@@ -31,6 +31,13 @@ func NewMsgServiceRouter() *MsgServiceRouter {
 // MsgServiceHandler defines a function type which handles Msg service message.
 type MsgServiceHandler = func(ctx sdk.Context, req sdk.Msg) (*sdk.Result, error)
 
+type IMsgServiceRouter interface {
```

Comment:
Generally public types have a comment ... has `go lint` or `go critic` been run on this?